### DISCUSSION TITLE
Suggestions for implementing an Action Sheet?

### DISCUSSION THREAD

**Discussion by boozedog** [Instawork/hyperview#644] [Q&A] [answered]
Interested in how I would go about implementing mobile client behavior such as an Action Sheet in Hyperview? e.g., https://ionicframework.com/docs/api/action-sheet

**Comment by adamstep** [comment, marked as answer]
Hi, would the existing alert action work for your needs? https://hyperview.org/docs/reference_behavior_alert

If not, you could create an equivalent behavior leveraging the library https://github.com/expo/react-native-action-sheet to use in your app. If you want to explore this, I can give some more pointers.

**Reply by boozedog** [reply]
Thanks for the thoughtful response @adamstep ! Both of these look like viable options. Will reach out when we get time to explore this more deeply.
